services/booking-api/lambdas/getAdministratorDetails.js

```diff
@@ -0,0 +1,36 @@
+import to from 'await-to-js';
+
+import * as response from '../../../libs/response';
+
+import booking from '../helpers/booking';
+
+export async function main(event) {
+  const { email } = event.pathParameters;
+
+  if (!email) {
+    return response.failure({
+      status: 400,
+      message: 'Missing required parameter: "email"',
+    });
+  }
+
+  const emailRegex = /.*@.*\..*/;
+  const emailIsValid = emailRegex.test(email);
+
+  if (!emailIsValid) {
+    return response.failure({
```

Comment:
Maybe this shoud be logged as well using our logger helper found in libs/log.js?
This can be applied to all places where we throw / return an error :)